### ISSUE TITLE
Change Contact us and Helpful VA phone numbers pages from tables to lists

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#30852]
## Issue Description
As part of our work to ensure our pages are accessible to all users, we need to change some of our tables to lists. The 2 pages to work on are:

- [Contact us](https://www.va.gov/contact-us/)
- [Helpful VA phone numbers](https://www.va.gov/resources/helpful-va-phone-numbers/)

When creating these lists, we need to check if there's any special HTML we need to use or if there's any other guidance to note from the accessibility repo found [here](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility).

Once we have reviewed how the lists should be created, we should confirm with the accessibility team before reworking the pages.

## Tasks
- [x] Review accessibility guidance for creating lists in accessibility repo (Selina)
- [x] Confirm with accessibility team we are building lists correctly (Selina/Randi)
- [x] Rework Contact us page (Selina)
- [x] Rework Helpful VA phone numbers page (Selina)
- [x] Review and publish pages (Randi)

## Acceptance Criteria
- [x] Contact us page uses lists instead of tables on live site
- [x] Helpful VA phone numbers page uses lists instead of tables on live site

**Comment by RLHecht** [comment]
Phone numbers page is now live. Closing ticket.
